multinode/bootstrapping.md

```diff
@@ -0,0 +1,190 @@
+# Data Node Bootstrapping [](distdb-bootstrap)
+
+Scaling up TimescaleDB is achieved by adding data nodes to an access
+node using the [`add_data_node`][add_data_node] and
+[`attach_data_node`][attach_data_node] functions. In order for a data
+node to be available to the access node it is necessary to ensure that
+it is properly setup to work as a data node. This process is referred
+to as *bootstrapping the data node*, which is done automatically each
+time a data node is added to an access node (unless `bootstrap` is
+false, in which case it just verifies that the data node is properly
+set up).
+
+The steps performed when boostrapping a data node are:
+
+1. Connect to the data node and authenticating the user.
+1. Check that there is a database on the data node and create it
```

Comment:
```suggestion
1. Check that there is the database on the data node and create it
```
If I don't mistake, the database should have a specific name, not an any database, right?